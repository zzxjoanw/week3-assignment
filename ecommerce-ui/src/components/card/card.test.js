import React from 'react';
import Card from './card';

describe('Card', () => {
  it('renders without crashing', () => {
    expect(<Card />).toMatchSnapshot();
  });
})