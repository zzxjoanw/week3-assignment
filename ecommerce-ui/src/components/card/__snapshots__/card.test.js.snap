// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders without crashing 1`] = `<Card />`;

exports[`renders without crashing 1`] = `<Card />`;
